mantidproject/mantid
# Improvements to SimulatedDensityOfStates

Fixes #13425

To test:
- Run in `DOS` mode with `squaricn.phonon`
- See that there is a small "stick diagram" of the un-broadened peaks in the second histogram, this is used to show the locations of the contributions to a broad peak
- Run in `BondTable` mode with `squaricn.castep`
- See that the parsed table matches the contents of the file

Release notes updates [here](http://www.mantidproject.org/index.php?title=ReleaseNotes_3_5_Framework_Changes&diff=25044&oldid=24984).
